indico/modules/designer/forms.py

```diff
@@ -20,3 +21,9 @@ class AddTemplateForm(IndicoForm):
     type = IndicoEnumSelectField(_('Template'), enum=TemplateType, default=TemplateType.poster)
     is_clonable = BooleanField(_('Allow cloning'), widget=SwitchWidget(), default=True,
                                description=_('Allow cloning this template in subcategories and events'))
+
+
+class CloneTemplateForm(IndicoForm):
+    category = CategoryField(_('Category'), [DataRequired()], require_event_creation_rights=True,
```

Comment:
This should require full management rights, not just event creation rights.

Reply:
The CategoryField only takes in a `require_event_creation_rights` param, should I add another one that checks for management rights to the field or do the validation using a func `validate_category()`?

Reply:
Yes, you'll need to add this option and pass it along in various places until you get into the category picker where it's actually used to determine whether a category can be selected or not...

Reply:
Okay